cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
@contolini @anselmbradford We can't set defaults in a `.env` file but I think what Ans is getting at is that it should be really clear what our environment variables are at any given point by examining the environment `os.environ`. So rather than fall back on a value if the environment is not set, it would be better if we _set_ the environment values with the defaults. It could be done in this file.